src/lib/action/field-rename.ts

```diff
@@ -29,11 +29,15 @@ class FieldRenameAction extends EntityAction {
     const fields = ct.fields
     const field = fields.getField(this.fieldId)
 
+    const fieldAnnotations = ct.getFieldAnnotations(this.fieldId)
     fields.deleteField(this.fieldId)
     field.id = this.props.newId
     delete field.newId
 
     fields.setField(this.props.newId, field)
+    if (fieldAnnotations) {
```

Comment:
Can’t hurt to add a test for this as well. E.g. so it doesn’t get overlooked in a refactoring.

Reply:
Is this it https://github.com/contentful/contentful-migration/pull/1060/files#diff-72b02b4b4284f14f9bfdfe3de28700e0469eb4d53066c70719c59a89846f3040 ? :)

Reply:
Yes, that works. I had only seen the examples up to this point.